shared/scss/options.scss

```diff
@@ -19,3 +19,13 @@
     margin-bottom: 10px;
     color: $slate;
 }
+
+.options-content__privacy {
```

Comment:
what's up with all those css changes 🤔 are those related to this task?

Reply:
Yeah, it's for the settings page. There are some inconsistencies in the spacing there where the actual design didn't look at all like the mock up, so I tweaked a few things outside of the GPC section for consistency. There are plans to revamp this whole page, so there will be more seemingly unrelated css changes incoming shortly.